src/LocaliseInterface.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Part of the Joomla Framework Language Package
+ *
+ * @copyright  Copyright (C) 2005 - 2015 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE
+ */
+
+namespace Joomla\Language;
+
+/**
+ * Joomla Framework Language Interface
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+interface LocaliseInterface
```

Comment:
I guess the interface isn't essential. I just preferred it to an abstract class where in many languages you're going to override every method. The Abstract class is largely just a 'western' implementation (useful for en-GB)